sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/QueryKeywords.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Azure.DigitalTwins.Core.Queries.QueryBuilders
+{
+    /// <summary>
+    /// Query keywords for building queries in string format.
+    /// </summary>
+    internal static class QueryKeywords
```

Comment:
I would call this class `QueryConstants` and add a bunch of other fields here. like : 
`TOP`, `COUNT`, `RELATED` IS_OFMODEL`  ....